address.bib

```diff
@@ -72,6 +72,9 @@ @string{ Stanford_CA_USA
 @string{ FL_USA              = "FL" # comma # USA }
 @string{ BocaRaton_FL_USA    = "Boca Raton" # comma # FL_USA }
 
+@string{ IL_USA              = "IL" # comma # USA }
+@string{ Chicago_FL_USA    = "Chicago" # comma # IL_USA }
```

Comment:
the key states Chicago_**FL**_USA but the actual value is # comma # **IL**_USA ?